src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingRunnable.java

```diff
@@ -53,9 +53,11 @@ public void run() {
                 dos = new DataOutputStream(process.getOutputStream());
                 // Set home directory to data folder for syncthing to use.
                 dos.writeBytes("HOME=" + Environment.getExternalStorageDirectory() + " ");
-                dos.writeBytes("STTRACE=" + pm.getString("sttrace", "") + " ");
+                dos.writeBytes("STTRACE=" + sp.getString("sttrace", "") + " ");
                 dos.writeBytes("STNORESTART=1 ");
                 dos.writeBytes("STNOUPGRADE=1 ");
+                dos.writeBytes("STGUIAUTH='" + sp.getString("gui_user", "") + ":" +
```

Comment:
As this is sensitive information, it should be moved to the Runtime.exec function, see http://docs.oracle.com/javase/6/docs/api/java/lang/Runtime.html#exec%28java.lang.String,%20java.lang.String[]%29